src/main/resources/de/thetaphi/forbiddenapis/signatures/commons-io-unsafe-2.15.1.txt

```diff
@@ -0,0 +1,17 @@
+commons-io-unsafe-2.14.0.txt# (C) Copyright Uwe Schindler (Generics Policeman) and others.
```

Comment:
Same here, having the filename at the start of the first line looks incorrect

Reply:
Copy paste bug :)